docs/docs/core/hooks/use-fetch-client.mdx

```diff
@@ -0,0 +1,102 @@
+                }
+            </div>
+            <button onClick={handleGetData}>Show Items</button>
+        </div>
+    )
+}
+```
+
+## Methods
+
+Essentially, this is an abstraction around the axios instance wrapped in a hook. It provides a simple interface to handle API calls to the Strapi backend.
+It handles request cancellation under the hook with [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController), and when the component gets unmounted all the reqeusts that it has made are aborted.
+
+The hook exposes four methods:
+
+- **get(url, config)**: which is a method that given a relative url (string), will make a GET request to the Strapi backend with the configuration (object) passed as second parameter, if needed.
+- **post(url, data, config)**: which is a method that given a relative url (string), will make a POST request with the data (object) provided to the Strapi backend with the configuration passed as third parameter (object), if needed.
+- **put(url, data, config)**: which is a method that given a relative url (string), will make a PUT request with the data (object) provided to the Strapi backend with the configuration passed as third parameter (object), if needed.
+- **del(url, config)**: which is a method that given a relative url (string), will make a DELETE request to the Strapi backend with the configuration passed as second parameter (object), if needed.
+
+## Implementation details
+
+The fetch instance is running axios under the hook with two request interceptors.
+
+### Base URL
+
+The default URL will be the one defined in the environment variable: `STRAPI_ADMIN_BACKEND_URL`.
+
+### Interceptors
+
+#### Request
+
+The request interceptor adds the following parameters to the header:
+
+```js
+{
+    Authorization: `Bearer <AUTH_TOKEN>`,
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+}
+```
+
+#### Response
+
+If everything has gone correctly, the response is returned as it comes from the backend. However, if it cointains a **status code of 401**, the authentication details will be removed from
+the application storage and the window will be reload
+
+:::caution
+Have this in mind if using this hook in pages where the auth token is not available, such as login, because it will create an infinite loop if not treating the error correctly. See section [Troubleshooting](#troubleshooting)
+:::
+
+## Further Reading
+
+- [axios instance API](https://axios-http.com/docs/instance)
+- [AbortController](https://axios-http.com/docs/cancellation)
+
+## Troubleshooting
+
+### Authentication problems
+
+Trying to access a protected route from a context where the auth token is not available may lead to an infinite loop, because of the response interceptor that
+reloads the page when obtaining a 401 response. One option available to avoid this from happening is to not considering a 401 response as an error, as follows:
```

Comment:
```suggestion
reloads the page when obtaining a 401 response. One option to avoid this from happening is to not consider a 401 response as an error, see below:
```